test/client_test.exs

```diff
@@ -426,7 +299,7 @@ defmodule ClientTest do
         :connect,
         fn ^address,
            ^port,
-           [:binary, {:packet, 0}, {:active, false}, {:send_timeout, 60_000}],
+           [:binary, {:packet, 0}, {:active, true}, {:send_timeout, 60_000}],
```

Comment:
This makes perfect sense to me. Most use cases around this are not subject to copious amounts of requests, however we may want to think about it a little 🤔

Reply:
Thought about this some... I think active once will be needed for some use cases, but we can defer to that until it is actually needed. It's pretty much add the complexity now or later :)